XamlControlsGallery/App.xaml

```diff
@@ -40,6 +40,12 @@
                 </ResourceDictionary>
             </ResourceDictionary.ThemeDictionaries>
 
+            <LinearGradientBrush x:Key="HeroImageGradientBrush" StartPoint="0.5,0" EndPoint="0.5,1">
+                <GradientStop Offset="0" Color="{ThemeResource SystemAltMediumColor}"/>
+                <GradientStop Offset="0.3" Color="{ThemeResource SystemAltMediumColor}"/>
```

Comment:
Maybe we could change the 0.3 to 0.5 so the "dark top border" of the hero image is high enough that it sort of resembles the standard border apps have. This way, in my opinion, it would blend in a bit more natural as the bottom border of that "top border" would line up nicely with the caption controls and the app name on the left. 

Current behaviour:
![image](https://user-images.githubusercontent.com/16122379/64386881-98198280-d03a-11e9-8bb1-f8fb677ecc0a.png) ![image](https://user-images.githubusercontent.com/16122379/64386959-dca51e00-d03a-11e9-9a36-cf7931556434.png)

With 0.5 as GradientStop:
![image](https://user-images.githubusercontent.com/16122379/64387022-08280880-d03b-11e9-8b46-828d78ca6cc2.png)  ![image](https://user-images.githubusercontent.com/16122379/64387069-2ee63f00-d03b-11e9-98f8-f7a4daf43a45.png)

Of course this however boils down to personal preference. Either way this is a really nice touch to the hero image!